concent_api/conductor/models.py

```diff
@@ -94,6 +117,9 @@ class OutputFormat(ChoiceEnum):
     # Source code of the Python script to be executed by Blender.
     blender_crop_script = TextField(blank=True, null=True)
 
+    # Parameters needed to generate Python script to be executed by Blender.
+    blender_crop_script_parameters = OneToOneField(BlenderCropScriptParameters, related_name='blender_subtask_definition', blank=True, null=True)
```

Comment:
Same here. I'd put fields from `BlenderCropScriptParameters` directly here.

Reply:
Hmm, but this would extend `BlenderSubtaskDefinition` table too much I think that script parameters could be independent table. I changed that now `BlenderCropScriptParameters` is merged all in one, but I'd stay with this parameters in one separate table instead keeping all of them i `BlenderSubtaskDefinition` model